src/NHibernate/Event/Default/DefaultMergeEventListener.cs

```diff
@@ -71,8 +81,19 @@ public virtual void OnMerge(MergeEvent @event)
 							transientEntityName,
 							transientEntity.ToString());
 					}
+
+					// create a comma-delimited list of the entity names that failed to merge
```

Comment:
Hey Alex,
I'm not sure what to do next about this pull request. All of the platform-specific stuff that was failing the DB-specific tests has now been ironed-out, and yet some of the checks are still failing for unrelated reasons. Are there any further steps required on my part?